log.go

```diff
@@ -89,30 +89,29 @@ var subsystemLoggers = map[string]slog.Logger{
 // initLogRotator initializes the logging rotater to write logs to logFile and
 // create roll files in the same directory.  It must be called before the
 // package-global log rotater variables are used.
-func initLogRotator(logFile string) {
+func initLogRotator(logFile string) error {
 	logDir, _ := filepath.Split(logFile)
 	err := os.MkdirAll(logDir, 0700)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to create log directory: %v\n", err)
-		os.Exit(1)
+		return errors.Errorf("failed to create log directory: %v", err)
 	}
 	r, err := rotator.New(logFile, 10*1024, false, 3)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to create file rotator: %v\n", err)
-		os.Exit(1)
+		return errors.Errorf("failed to create file rotator: %v", err)
 	}
 
 	logRotator = r
+	return nil
 }
 
 // RegisterLogger should be called before logRotator is initialized.
 func RegisterLogger(tag string) (slog.Logger, error) {
 	if logRotator != nil {
-		return nil, fmt.Errorf("cannot register logger after log rotator is initialized")
+		return nil, errors.E("cannot register logger after log rotator is initialized")
```

Comment:
Again, why not use `fmt`?

Reply:
github.com/decred/dcrwallet/errors is used all through the project

Reply:
alright

Reply:
But can't we use concrete error types? Searching the error strings for the exact error isn't that nice.